### PR TITLE
added hands test file

### PR DESCRIPTION
**Pull request by cabanier**
This example shows basic use of the WebXR hands API.
If the device supports the feature, it will render your hands as a set of cubes.

### CODE REVIEW

**Comment by cabanier** [comment]
FYI Oculus browser 10.2 supports this feature.
chrome://flags -> WebXR experiences with joint tracking -> set to 'Enabled' -> restart

**Comment by cabanier** [comment]
@toji Can this be merged?

File: immersive-hands.html [hunk 1]
```diff
@@ -0,0 +1,346 @@
+<!doctype html>
+<!--
+Copyright 2018 The Immersive Web Community Group
```
Comment (toji) [inline immersive-hands.html:3]:
```suggestion
Copyright 2020 The Immersive Web Community Group
```

**Review by toji — CHANGES_REQUESTED** [review]
Apologies for not noticing this earlier. The sample looks good, but given that the API is still technically a WIP could we list this sample on the "Proposals" tab instead of the main sample list? At least until the API is more formally shipping.

The proposals tab actually just got re-enabled today by a different set of samples that @bialpio added, so it should be a pretty minor thing to move it over.

**Comment by cabanier** [comment]
> Apologies for not noticing this earlier. The sample looks good, but given that the API is still technically a WIP could we list this sample on the "Proposals" tab instead of the main sample list? At least until the API is more formally shipping.

No problem!

**Comment by toji** [comment]
You'll need to move the test html file to the proposals folder as well, and make sure that any resources that it links to are updated to the right relative paths.

**Review by toji — APPROVED** [review]
Looks good, thanks! Github says there's a conflict, but we're good to merge once that's resolved.

**Comment by cabanier** [comment]
There is no conflict :-)

**Pull request merged**